README.md

```diff
@@ -136,11 +136,14 @@ check out [our style guide](doc/internal/code-style.markdown).
 
 You'll probably want to run the tests; here's how:
 
-	$ make -j5 PROFILE=1 all test  # 5 is CPU cores + 1, to parallelize the build
-	$ (cd test && TMPDIR=/dev/shm ./test --order rand) && cargo test
-
-Note the use of ramdisk as `TMPDIR`: some tests create temporary files, which
-slows them down if `TMPDIR` is on HDD or even SSD.
+	$ TMPDIR=/dev/shm make -j5 PROFILE=1 check
+
+The "5" here is the number of CPU cores in your machine *plus one*. This
+parallelises the build. Rust tests already utilize as many cores as they can,
+but if you want to limit them, use `RUST_TEST_THREADS` environment variable.
```

Comment:
I reckon there should to be an article: …use **the** `RUST_TEST_THREADS` environment variable.

Reply:
Thanks! Fixed in 9685313d961ffa756165f8fce37a572c0fd85f56.